### DISCUSSION TITLE
AmazonSQSBufferedAsyncClient ?

### DISCUSSION THREAD

**Discussion by dwalend** [laserdisc-io/fs2-aws#761] [Q&A] [answered]
Does fs2-aws support use of AmazonSQSBufferedAsyncClient ? (If not - are there good reasons I should avoid that thing?)

And - is this the right place for fs2-aws discussions or is there a better forum?

Thanks,

David

**Comment by semenodm** [comment]
this is right place for discussions. thank you for your interest in this library.
the library designed in the way that you configure the AWS SDK client outside and then provide to the stream. So i would say you should try to use any client you need. 
unless this specific client has some specific api that affects how we pull sqs messages from the queue

**Comment by semenodm** [comment, marked as answer]
oh, i just realized that AmazonSQSBufferedAsyncClient is in AWS SDK V1. we do not support it, only V2
